src/irmin-pack/layered/irmin_pack_layered.ml

```diff
@@ -387,18 +329,19 @@ struct
         { throttle; state = `None; lock = Lwt_mutex.create () }
 
       let v config =
+        let module Config = Config_layered in
```

Comment:
Maybe don't rename the module here, as you use `Config_layered` in the `migration` as is.